src/bidiMapper/domains/script/channelProxy.ts

```diff
@@ -171,64 +171,64 @@ export class ChannelProxy {
+            functionDeclaration: String(
+              async (channelHandle: {getMessage: () => Promise<unknown>}) =>
+                channelHandle.getMessage()
+            ),
+            arguments: [
+              {
+                objectId: channelHandle,
+              },
+            ],
+            awaitPromise: true,
+            executionContextId: realm.executionContextId,
+            serializationOptions: {
+              serialization:
+                Protocol.Runtime.SerializationOptionsSerialization.Deep,
+              ...(this.#properties.serializationOptions?.maxObjectDepth ===
+                undefined ||
+              this.#properties.serializationOptions.maxObjectDepth === null
+                ? {}
+                : {
+                    maxDepth:
+                      this.#properties.serializationOptions.maxObjectDepth,
+                  }),
             },
-          ],
-          awaitPromise: true,
-          executionContextId: realm.executionContextId,
-          serializationOptions: {
-            serialization:
-              Protocol.Runtime.SerializationOptionsSerialization.Deep,
-            ...(this.#properties.serializationOptions?.maxObjectDepth ===
-              undefined ||
-            this.#properties.serializationOptions.maxObjectDepth === null
-              ? {}
-              : {
-                  maxDepth:
-                    this.#properties.serializationOptions.maxObjectDepth,
-                }),
-          },
-        }
-      );
+          }
+        );
 
-      if (message.exceptionDetails) {
-        // TODO: add logging.
-        // TODO: check if a error should be thrown.
-        return;
-      }
+        if (message.exceptionDetails) {
+          // TODO: add logging.
+          // TODO: check if a error should be thrown.
+          return;
+        }
 
-      eventManager.registerEvent(
-        {
-          type: 'event',
-          method: ChromiumBidi.Script.EventNames.MessageEvent,
-          params: {
-            channel: this.#properties.channel,
-            data: realm.cdpToBidiValue(
-              message,
-              this.#properties.ownership ?? Script.ResultOwnership.None
-            ),
-            source: {
-              realm: realm.realmId,
-              context: realm.browsingContextId,
+        eventManager.registerEvent(
+          {
+            type: 'event',
+            method: ChromiumBidi.Script.EventNames.MessageEvent,
+            params: {
+              channel: this.#properties.channel,
+              data: realm.cdpToBidiValue(
+                message,
+                this.#properties.ownership ?? Script.ResultOwnership.None
+              ),
+              source: {
+                realm: realm.realmId,
+                context: realm.browsingContextId,
+              },
             },
           },
-        },
-        realm.browsingContextId
-      );
-    }
+          realm.browsingContextId
+        );
+      }
+    } catch {}
```

Comment:
Something is missing here, either some or all of those:

- Rationale for the PR (what problem does it resolve?) --> could just augment the commit message; alternatively (or as well), add a code comment here
- Log the `catch`?
- Why was this comment removed?

```
 // TODO(#294): Remove this loop after the realm is destroyed.
 // Rely on the CDP throwing exception in such a case.
```

Reply:
Done.